openstack/sharedfilesystems/apiversions/errors.go

```diff
@@ -0,0 +1,23 @@
+package apiversions
+
+import (
+	"fmt"
+)
+
+// ErrVersionNotFound is the error when the requested API version
+// could not be found.
+type ErrVersionNotFound struct{}
+
+func (e ErrVersionNotFound) Error() string {
```

Comment:
> There is a kind of framework for errors in the gophercloud library.

I am aware of this.

> However, the framework is not used here so it feels inconsistent.

Can you provide a more concrete example of how it can be used? I did not see an existing error type that would best fit the errors defined here and I didn't want to use `BaseError` just to say I used `BaseError`.